modules/ROOT/pages/to-share-an-asset.adoc

```diff
@@ -0,0 +1,63 @@
+Or click a filter to list only users who have Viewer, Contributor, or Admin rights.
+. Click Add.
+. Repeat steps 2 and 3 to add additional users to share the asset with.
+
+You can also use the Share menu to change who you are sharing the asset with or to stop sharing with one or more users.
+
+== Share With Your Organization
+
+Exchange administrators can grant viewer access for an asset to everyone in their organization.
+
+. Click an asset to show its details and click Share.
+. In the Share menu, begin typing the name of your organization.
+. Select the menu item for everyone at your organization.
+
+When an asset is shared with an organization, email notification is sent to the maintainer of the organization's default "All Users" team. Usually this is the organization administrator.
+
+You can also use the Share menu to stop sharing the asset with everyone at the organization.
+
+Any Exchange user can click the *Shared with me* link in the left navigation menu of Exchange to show all assets shared with their user account and all assets shared with their organization.
+
+== Share With an External Organization
+
+Exchange administrators can grant viewer access for an asset to everyone in a chosen external organization.
+
+The organization granting viewer access is the granting organization, and the organization that receives access is the receiving organization.
+
+. The receiving organization adds the granting organization to its "External Access" list in Anypoint Access Management, as described in xref:access-management::external-org-access.adoc[External Organization Access].
++
+Assets can only be shared with another organization if that organization has enabled receiving viewer access to assets from external organizations.
++
+. Login to your Exchange administrator account in the granting organization.
+. Click an asset to show its details and click Share.
+. In the Share menu, begin typing the name of the external organization you want to share the asset with.
+. Select the organization you want to grant viewer access to.
+. Repeat steps 4 and 5 to add additional organizations to share the asset with.
+
+When an asset is shared with an organization, email notification is sent to the maintainer of the organization's default "All Users" team. Usually this is the organization administrator.
+
+You can also use the Share menu to change who you are sharing the asset with or to stop sharing with one or more organizations.
+
+Any Exchange user can click the *Shared with me* link in the left navigation menu of Exchange to show all assets shared with their user account and all assets shared with their organization.
+
+Only internal users can request access to an API or gain the contributor or administrator roles. To grant these permissions to a user from another organization, invite the user to become an internal user of your organization, and then grant the contributor, administrator, or API access requestor permissions.
```

Comment:
It's not possible to invite a user from another organization to join your own organization? Or it's not possible for a user in your own organization to get contributor, admin, or API access requestor permissions?

Reply:
Sorry, I double-check that, you can invite a user from another organization to join your own organization. But at this point, the user will be moved from his organization to your organization. For the organization, "A", It is not worth to lose a user, just because the organization "B" wants to share him an asset.

I would not write it, I do not think it brings much value

Reply:
True, removing a user from the other org to join your org wouldn't be worth it. Can the user belong to both orgs?

Reply:
As far as I know, no he cannot.

Reply:
Sara confirmed that is correct. Thank you for correcting that! I removed the part about granting these perms to a user from another org.